src/index.html

```diff
@@ -21,7 +24,389 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="product-card"
+        data-qa="card"
+      >
+        <div class="product-card__block">
+          <img
+            src="./images/imac.jpeg"
+            alt="product card image"
+            class="product-card__image"
+          />
+        </div>
+
+        <div class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product-card__product-code">Product code: 195434</div>
+
+        <div class="product-card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="product-card__reviews-count">Reviews: 5</div>
+        </div>
+
+        <div class="product-card__price-block">
+          <p class="product-card__price-text">Price:</p>
+
+          <p class="product-card__price">$2,199</p>
+        </div>
+
+        <button
```

Comment:
The button with the class 'product-card__button' has an attribute 'data-qa="card-hover"' only in the first occurrence (line 153). According to the task checklist, if this attribute is required for testing or other purposes, it should be consistently present on all similar elements or omitted if not necessary.